### ISSUE TITLE
Latex view in IRKernel

### ISSUE THREAD

**Issue by isuruf** [symengine/symengine.R#47]
For example, Python and C++ notebooks will use `latex` function to convert to latex and display a symengine object.

**Comment by Marlin-Na** [comment]
I have thought about supporting latex printing in R markdown document, and it should also be possible in Jupyter notebook. It seems that there is support for latex string generation in the C++ library but not yet wrapped in cwrapper, right? Do you think we should use SymEngine's latex printer or implement R's?

**Comment by isuruf** [comment]
> It seems that there is support for latex string generation in the C++ library but not yet wrapped in cwrapper, right?

Yes. SymEngine's latex printer is at its early stages and there's much to improve. It's faster since it uses symengine internals. If there are any issues with it, we can fix it and it would benefit the python and other wrappers.

**Comment by Marlin-Na** [comment]
Generating latex view in Rmarkdown document is now supported in Rmarkdown with https://github.com/symengine/symengine.R/pull/64.
